smithy-swift-codegen/src/test/resources/enum-shape-test.smithy

```diff
@@ -0,0 +1,26 @@
+$version: "2.0"
+
+namespace smithy.example
+
+service Example {
+    version: "1.0.0"
+    operations: [
+        ChangeCard
+    ]
+}
+
+operation ChangeCard {
+    input: Card
+    output: Card
+}
+
+structure Card {
+    suit: Suit
+}
+
+enum Suit {
```

Comment:
are these string based enum or int based?

Reply:
String based, the int enums follow a different syntax, e.g.,

```
intEnum Number {
    ACE   = 1
    TWO   = 2
    THREE = 3
    FOUR  = 4
    FIVE  = 5
    SIX   = 6
    SEVEN = 7
    EIGHT = 8
    NINE  = 9
    TEN   = 10
    JACK  = 11
    QUEEN = 12
    KING  = 13
}
```